packages/sui-react-web-vitals/src/index.js

```diff
@@ -102,16 +104,20 @@ export default function WebVitalsReporter({
       if (!isAllowed || !logger?.cwv || rating === RATING.GOOD) return
 
       const target = getTarget({name, attribution})
+      const {loadState, eventType} = attribution
 
       logger.cwv({
         name: `cwv.${name.toLowerCase()}`,
         amount,
         path: pathname,
-        ...(routeid && {routeId: routeid}),
         target,
-        loadState: attribution.loadState,
         visibilityState: document.visibilityState,
-        ...(attribution.eventType && {eventType: attribution.eventType})
+        ...(routeid && {routeId: routeid}),
+        ...(loadState && {loadState}),
+        ...(eventType && {eventType}),
+        ...(deviceMemory && {deviceMemory}),
+        ...(effectiveType && {effectiveType}),
```

Comment:
Maybe is more semantic to use effectiveConnection or something like that.

Reply:
IMHO, I'd rather stick to what the `window.navigator` API returns...

Reply:
> IMHO, I'd rather stick to what the `window.navigator` API returns...

Agree, that's my first thought